snaps/reference/cli/subcommands.md

```diff
@@ -29,6 +29,27 @@ Builds a Snap from source.
 
 `b` is an alias for `build`.
 
+#### `analyze`
+
+<Tabs>
+<TabItem value="Syntax">
+
+```bash
+yarn mm-snap build --analyze
+```
+
+</TabItem>
+</Tabs>
+
+Enables analyzing the Snap [bundle](../../learn/about-snaps/files.md#bundle-file).
+This uses [`webpack-bundle-analyzer`](https://github.com/webpack-contrib/webpack-bundle-analyzer) under the hood,
+which creates an interactive visualization of the contents of your bundle.
+The visualization is located at the port displayed in the command line output.
```

Comment:
Does this mean something like `http://localhost:<port>`>

Reply:
Yes - maybe "URL" is more clear?